index.html

```diff
@@ -238,6 +238,23 @@ <h3 class="lang-header">Languages</h3>
         </ul>
       </div>
     </section>
+    <section class="aboutme contact-me">
+      <div class="bio">
+        <img class="dot1" src="./icons/dot1.png" alt="" />
+        <h2 class="contactme-header">Contact me</h2>
+        <form action="https://formspree.io/f/mdovgdgq" method="POST">
+          <p class="contactme-p">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+          <input id="name" name="Name" type="text" class="form-control" value="Jedda" maxlength="30" required />
+          <input id="name" name="Email" type="email" class="form-control" value="fathiato@gmail.com" required />
+          <textarea name="text-area" id="mssg" class="form-control" placeholder="write your message here" maxlength="500" required></textarea>
+          <div>
+            <button class="getintouch">Get in touch</button>
+            <img class="rec" src="./icons/rec.png" alt="" />
+            <img class="dot2" src="./icons/dot2.png" alt="" />
```

Comment:
- [ ] If you notice the images in the contact form are purposely for decoration. Therefore they should be added using the CSS styling instead of `img`. Kindly ensure to fix this, so that the contact form is more professional and matches the Figma design ☺️ .